3drepo/3drepo.io
# ISSUE #4642 - Tickets supports transformations

This fixes #4642 

#### Description
- TIcket not longer throws an error while having a sequence opened
- TIcket get saved with the transformations
- Selecting the ticket shows the transformation

#### Test cases
- Create a ticket while having a sequence opened with some mesh moved 
- Enter the model and select the ticket, the mesh should be moved. Deselecting the ticket should reset the model back to normal
